### PR TITLE
GOV-359: Integration test for batch aggregate API

### PR DESCRIPTION
**Pull request by arkadasfynarfin**
## Description

[GOV-359](https://mifosforge.jira.com/browse/GOV-359)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-359]: https://mifosforge.jira.com/browse/GOV-359?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/test/java/resources/batch.feature [hunk 1]
```diff
@@ -14,6 +14,14 @@ Feature: Batch Details API test
     Then I should get non empty response
     And I should have "PollingPath" and "SuggestedCallbackSeconds" in response
 
+  Scenario: Batch aggregate API Test
+    Given I have the demo csv file "test.csv"
+    And I have tenant as "gorilla"
+    When I call the batch transactions endpoint with expected status of 200
```
Comment (fynmanoj) [inline src/test/java/resources/batch.feature:20]: expect 202

File: src/test/java/resources/batch.feature [hunk 2]
```diff
@@ -14,6 +14,14 @@ Feature: Batch Details API test
...
+    Then I should get non empty response
+    And I fetch batchId from response
+    When I call the batch aggregate API with expected status of 200
+    Then I should get non empty response
```
Comment (fynmanoj) [inline src/test/java/resources/batch.feature:24]: you should open and check the response

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/test/java/resources/batch.feature (hunk 2 above)
Comment (avikganguly01) [reply src/test/java/resources/batch.feature:24]: @arkadasfynarfin : Please address this review comment.

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: src/test/java/resources/batch.feature (hunk 1 above)
Comment (danishjamal104) [reply src/test/java/resources/batch.feature:20]: fix this

**Pull request closed without merging**